WordPress/Classes/ViewRelated/Likes/LikesListController.swift

```diff
@@ -97,16 +97,38 @@ class LikesListController: NSObject {
         // shows the loading cell and prevents double refresh.
         isLoadingContent = true
 
-        fetchLikes(success: { [weak self] users in
+        let successBlock: ([LikeUser]) -> Void = { [weak self] users in
             self?.likingUsers = users
             self?.isLoadingContent = false
-        }, failure: { [weak self] _ in
+        }
+
+        let failureBlock: (Error?) -> Void = { [weak self] _ in
             // TODO: Handle error state
             self?.isLoadingContent = false
-        })
+        }
+
+        fetchStoredLikes(completion: successBlock)
+        fetchLikes(success: successBlock, failure: failureBlock)
```

Comment:
I wonder if we should pass the same success block to both calls. The block will set isLoadingContent = false, which would happen when `fetchStsoredLikes` completes but while `fetchLikes` was still working.

Reply:
Maybe fetchedStoredLikes could just be an assignment, and the success block just be used by fetch likes?